tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Http/WebRequest/HttpWebRequest_EndGetResponse_Integration.cs

```diff
@@ -76,14 +78,25 @@ public class HttpWebRequest_EndGetResponse_Integration
 
                     if (scope is not null)
                     {
-                        if (setSamplingPriority && existingSpanContext?.SamplingPriority is not null)
+                        var span = scope.Span;
+                        var traceContext = span.Context.TraceContext;
+
+                        if (setSamplingPriority && traceContext != null)
                         {
-                            scope.Span.SetTraceSamplingPriority(existingSpanContext.SamplingPriority.Value);
+                            traceContext.SetSamplingPriority(existingSpanContext.SamplingPriority);
+
+                            // copy propagated tags
+                            var traceTags = TagPropagation.ParseHeader(existingSpanContext.PropagatedTags);
```

Comment:
As with GRPC, if these need to be sent in outgoing injected headers, this needs to be done earlier (in the `BeginGetResponse` instrumentation I think) 🤔 But I could be wrong, I find all this propagation stuff very confusing

Reply:
I believe Andrew's correct. We'll need to propagate the dm tag wherever `CreateInactiveOutboundHttpSpan(addToTraceContext: false)` is called. That should be in the following files:
- `tracer\src\Datadog.Trace\ClrProfiler\AutoInstrumentation\Http\WebRequest\HttpWebRequest_BeginGetRequestStream_Integration.cs`
- `tracer\src\Datadog.Trace\ClrProfiler\AutoInstrumentation\Http\WebRequest\HttpWebRequest_BeginGetResponse_Integration.cs`
- `tracer\src\Datadog.Trace\ClrProfiler\AutoInstrumentation\Http\WebRequest\HttpWebRequest_GetRequestStream_Integration.cs`

Reply:
Actually, since this tag is one of the PropgatedTags, I think it is already handled when we inject the inactive span into the headers

Reply:
Nice catch, thank you! I removed the code that copies that individual trace tags over.